signer/pgp/README.rst

```diff
@@ -0,0 +1,61 @@
+PGP Signing
+===========
+
+.. sectnum::
+.. contents:: Table of Contents
+
+This signer implements the Pretty Good Privacy signature format. It accepts data
+on the `/sign/data` interface and returns armored detached signatures.
+
+Configuration
+-------------
+
+Place a PGP Private Key in `autograph.yaml`:
+
+.. code:: yaml
+
+	signers:
+    - id: some-pgp-key
+      type: pgp
+      privatekey: |
+        -----BEGIN PGP PRIVATE KEY BLOCK-----
+
+        lQOYBFuW9xABCACzCLYHwgGba7hi+lwhD/Hr5qqpg+UuN+88NclYgLWyl1nPpx2D
+		...
+        HQASoA7mirON
+        =vJUu
+        -----END PGP PRIVATE KEY BLOCK-----
+
+Signature request
+-----------------
+
+This signer only supports the `/sign/data/` endpoint.
+
+.. code:: json
+
+	[
+		{
+			"input": "Y2FyaWJvdW1hdXJpY2UK",
+			"keyid": "some-pgp-key"
+		}
+	]
+
+Signature response
+------------------
+
+The response to a data signing request contains a PGP armored detached
```

Comment:
nit: "that contains" or "containing"

Reply:
sure about that one?

Reply:
no, too much coffee. Maybe "contains a PGP-armored detached signature"?

bleh

E_NESTED_NIT_LEVEL_TOO_HIGH